src/model/kaniOutputParser.ts

```diff
@@ -101,9 +101,20 @@ function parseChecksArray(checksArray: Array<string>): KaniResponse {
 	for (let i = 0; i < checksArray.length; i++) {
 		const checkInstance: string[] = checksArray[i].split('\n');
 		const checkInstanceObject: CheckInstance = convertChecktoObject(checkInstance);
-		if (checkInstanceObject.status == 'FAILURE') {
+		if (
+			checkInstanceObject.status == 'FAILURE' ||
+			checkInstanceObject.status == 'UNDETERMINED' ||
+			checkInstanceObject.status == 'UNREACHABLE' ||
+			checkInstanceObject.status == 'SATISFIED' ||
+			checkInstanceObject.status == 'UNSATISFIABLE'
+		) {
 			failureResponseMessage += checkInstanceObject.createFailureMessage() + '\n';
 			failureDisplayMessage = checkInstanceObject.createDisplayMessage() + '\n';
+		} else if (checkInstanceObject.status != 'SUCCESS') {
+			failureResponseMessage +=
+				checkInstanceObject.createFailureMessage() +
+				'WARNING: unknown status returned from Kani.\n\n';
```

Comment:
Suggest that you include the value of `checkInstanceObject.status` in the error message?

Reply:
Yes, the value of `checkInstanceObject.status` is already included in `checkInstanceObject.createFailureMessage()`.